### DISCUSSION TITLE
the solution posted is not working (Github pages isn't working after switching from public to private and then back to public)

### DISCUSSION THREAD

**Discussion by sjl4365** [community/community#140937] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've already tried a solution that was posted a while back  
1 Go to the ‘repository page’
2 Click on ‘settings’ 
3 Scroll down to ‘Github Pages’
4 Click on the dropdown for ‘Source’
5 Select ‘master branch’

Right now my Page for the project is set up like this
Under Build and deployment Source: Deploy from a branch
Under Branch: main root

**Comment by sjl4365** [comment]
https://github.com/orgs/community/discussions/23647. 
In this discussion someone posted the solution above but it does not seem to be working

**Reply by austinbediako** [reply, marked as answer]
First of all re change it to public  then refresh your browser.
go to the repo's settings and click on the pages link 
where you see 
![image](https://github.com/user-attachments/assets/7cd7bc52-1e98-4d67-b4c7-bf6b00022e99)
Build and deployments select build from branch
go to the branch and select the main production branch and save 
reload your browser and go to code 
![image](https://github.com/user-attachments/assets/a9312c00-d0b9-4c23-a65c-eb4d4ebb99ea)
you will see that yellow dot meaning your page is being deployed remember it only works if you have a static web file in the directory 
go back to the pages section in the repo's settings to find the link to your deployed page
